README.md

```diff
@@ -66,4 +76,4 @@ docker-compose down -v
 
 ### Running the Integration Tests
 
-TO BE COMPLETED
+Start the application as decribed in [running the application](#running-the-application) and run the integration tests using your preferred test runner.
```

Comment:
It might be worth specifying where the integration tests live, as I don't think that's mentioned anywhere in the file, but as that's not part of the logout change I will let it slide for this review